### PR TITLE
test: 100% coverage

### LINKED ISSUE #18: 100% code coverage
As titled, this module would benefit for 100% code coverage

### PR DESCRIPTION
**Pull request by simoneb**
Fixes #18

- remove some defaults checks which are unnecessary
- remove cache as it's never used
- add 100 flag to taprc

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: SerializerManager.js [hunk 1]
```diff
@@ -14,20 +14,15 @@ class Serializer {
 class SerializerManager {
   constructor (configuration) {
     this.serializers = configuration.serializers
-    this.cache = {}
   }
 
   findSerializer (types) {
-    const cacheValue = this.cache[types]
```
Comment (Eomm) [inline SerializerManager.js:21]: how this is never used?

Comment (simoneb) [reply SerializerManager.js:21]: Explained in a comment to the linked issue

Comment (Eomm) [reply SerializerManager.js:21]: Thks, I re-read now

The cache is broken
https://github.com/fastify/fastify-accepts-serializer/issues/30

**Comment by mcollina** [comment]
I would prefer the cache to be fixed

**Comment by simoneb** [comment]
> I would prefer the cache to be fixed

I can agree and it's tracked in #30, but it has nothing to do with this PR. The cache was already broken, this PR simply shows that it was.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**